packages/rewards/src/index.ts

```diff
@@ -56,16 +56,18 @@ export function calculateRewardsForUsers(
     );
   }
 
-  if (prizePoolDecayRate <= 0.1 || prizePoolDecayRate >= 0.9) {
+  if (prizePoolDecayRate < 0.1 || prizePoolDecayRate > 0.9) {
     throw new Error(
       `Invalid prize pool decay rate: ${prizePoolDecayRate}. Must be between 0 and 1.`,
     );
   }
 
-  if (boostTimeDecayRate <= 0.1 || boostTimeDecayRate >= 0.9) {
-    throw new Error(
-      `Invalid boost time decay rate: ${boostTimeDecayRate}. Must be between 0 and 1.`,
-    );
+  if (boostTimeDecayRate) {
```

Comment:
if it is provided we check the range. i relaxed the range a bit by removing the =